src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Calculate, Draw {
```

Comment:
Don't use verbs for class/interface names. Consider renaming `Calculate` and `Draw` to noun forms that represent their capabilities, such as `AreaCalculator` and `Drawable`, to better reflect their purpose.